Won't build on React Native 0.69

Hey folks. I upgraded my app's codebase from React Native 0.68 to 0.69 and now whenever I try and build the Android app, I get a bunch of errors that appear to relate to this library.
```
> Task :sparkfabrik_react-native-idfa-aaid:compileDebugKotlin FAILED
e: Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to suppress errors
e: /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/[redacted]/.gradle/caches/transforms-3/20245555e15bde9a3f3dfa3bcc495e08/transformed/jetified-kotlin-stdlib-jdk7-1.6.10.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/[redacted]/.gradle/caches/transforms-3/73e09de52d7bf9ac10986d5ba33618a1/transformed/jetified-kotlin-stdlib-common-1.6.10.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/[redacted]/.gradle/caches/transforms-3/9b14255b91bc8a9d35b5741a40c00d2f/transformed/jetified-kotlin-stdlib-jdk8-1.6.10.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/[redacted]/.gradle/caches/transforms-3/bb47c1b9d70583f10f2a780e89f78a4c/transformed/jetified-react-native-0.69.4-debug-api.jar!/META-INF/ReactAndroid_debug.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (20, 47): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (25, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (26, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (29, 21): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (31, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/[redacted]/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidPackage.kt: (18, 16): Unresolved reference: emptyList

/Users/[redacted]/.gradle/caches/transforms-3/0af99225e17e9f8ff15bd00e808923d6/transformed/jetified-kotlin-stdlib-1.6.10.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.

/Users/[redacted].gradle/caches/transforms-3/20245555e15bde9a3f3dfa3bcc495e08/transformed/jetified-kotlin-stdlib-jdk7-1.6.10.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.

/Users/[redacted]/.gradle/caches/transforms-3/73e09de52d7bf9ac10986d5ba33618a1/transformed/jetified-kotlin-stdlib-common-1.6.10.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.

/Users/[redacted]/.gradle/caches/transforms-3/9b14255b91bc8a9d35b5741a40c00d2f/transformed/jetified-kotlin-stdlib-jdk8-1.6.10.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.

/Users/[redacted]/.gradle/caches/transforms-3/bb47c1b9d70583f10f2a780e89f78a4c/transformed/jetified-react-native-0.69.4-debug-api.jar!/META-INF/ReactAndroid_debug.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
```
I've tried all the usual clearing of cache/build folders, invalidating caches etc.

Any idea what the issue might be?
Thanks

I'm also experiencing this issue, when I run `assembleDebug` I get this error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sparkfabrik_react-native-idfa-aaid:compileDebugKotlin'.

```
<details><summary>Click to see full error</summary>
```
> Task :sparkfabrik_react-native-idfa-aaid:compileDebugKotlin FAILED
e: Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to suppress errors
e: /Users/nxc/.gradle/caches/transforms-3/63eb694af5911e13964b8293df181ff6/transformed/jetified-kotlin-stdlib-jdk8-1.6.10.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/nxc/.gradle/caches/transforms-3/dafd79a392ae72d634de61010debede6/transformed/jetified-react-native-0.69.5-debug-api.jar!/META-INF/ReactAndroid_debug.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/nxc/.gradle/caches/transforms-3/f7bdf5baf8caaeeeabcaea3b82f99be8/transformed/jetified-kotlin-stdlib-jdk7-1.6.10.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/nxc/.gradle/caches/transforms-3/fa72ebd6eeb8460192ed1aa7806734af/transformed/jetified-kotlin-stdlib-common-1.6.10.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (20, 47): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (25, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (26, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (29, 21): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidModule.kt: (31, 17): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
The class is loaded from /Users/nxc/.gradle/caches/transforms-3/70ec13bb44dfe1062d524a1dd9fdeb4c/transformed/jetified-kotlin-stdlib-1.6.10.jar!/kotlin/Unit.class
e: /Users/nxc/ghq/github.com/app/node_modules/@sparkfabrik/react-native-idfa-aaid/android/src/main/java/com/sparkfabrikreactnativeidfaaaid/ReactNativeIdfaAaidPackage.kt: (18, 16): Unresolved reference: emptyList
w: Detected multiple Kotlin daemon sessions at build/kotlin/sessions
```
</details>

--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -1,4 +1,4 @@
-ReactNativeIdfaAaid_kotlinVersion=1.3.50
+ReactNativeIdfaAaid_kotlinVersion=1.6.0
 ReactNativeIdfaAaid_compileSdkVersion=28
 ReactNativeIdfaAaid_buildToolsVersion=28.0.3
 ReactNativeIdfaAaid_targetSdkVersion=28

--- a/package.json
+++ b/package.json
@@ -27,7 +27,8 @@
     "release": "release-it",
     "example": "yarn --cwd example",
     "pods": "cd example && pod-install --quiet",
-    "bootstrap": "yarn example && yarn && yarn pods"
+    "bootstrap": "yarn example && yarn && yarn pods",
+    "postinstall": "husky install"
   },
   "keywords": [
     "react-native",

